editor/editor_run.h

```diff
@@ -32,6 +32,9 @@
 #define EDITOR_RUN_H
 
 #include "core/os/os.h"
+#include "servers/display_server.h"
```

Comment:
What's this include for?

Reply:
Done, it was a relic from some old code.